### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by samlorlah**
**Linters, .gitignore and ReadMe**

- Html-CSS Linters was added and commit with a branch and comment

- .gitignore was added with node_modules as folder to ignore

**HTML**
Html Page was added and 2 sections were added. The header for the logo and Nav Hamburger and the first section of the website

**Stylesheet**
Stylesheet file was added in asset/css and the 2 sections were styled according to the template figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+
+    <link rel="stylesheet" href="./asset/css/style.css">
+</head>
+<body>
+    <header>
+        <div>
+            <span id="logo">My Logo</span>
```
Comment (vechicin) [inline index.html]: This content should be wrapped in an `<a>` tag. Please fix it.

Screen Shot 2022-06-29 at 18 15 23

File: asset/css/style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    padding-right: 5px;
+  }
+
+  #main-section {
+    background-image: url(../images/background.png);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: 100% 95%;
+    height: 588px;
+  }
```
Comment (vechicin) [inline asset/css/style.css]: Your background image placement is not quite right. Please take a closer look at the Figma design and fix it.

Screen Shot 2022-06-29 at 18 18 26

Additionally, you're missing the `border-radius` in the lower-left corner of the section, please add it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            <div id="connect">
+                <h3>Let's Connect</h3>
+            </div>
+            <div class="social-media-icons">
+                <a href="#"><img src="./asset/images/Vectortwitter.svg" alt="twitter"></a>
+                <a href="#"><img src="./asset/images/linkedin.svg" alt="linkedin"></a>
+                <a href="#"><img src="./asset/images/Vectorm-icon.svg" alt="M"></a>
+                <a href="#"><img src="./asset/images/Vectorgithub.svg" alt="github"></a>
+                <a href="#"><img src="./asset/images/Vectordribble.svg" alt="dribble"></a>
+            </div>
```
Comment (vechicin) [inline index.html]: This content should be wrapped in several `<li>` all of them wrapped in a `<ul>`. Please fix it.

Screen Shot 2022-06-29 at 18 19 59

File: asset/css/style.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+/* Mobile View */
+@media all and (max-width: 375px) {
+  * {
+    padding-left: 5px;
+    padding-right: 5px;
+  }
```
Comment (vechicin) [inline asset/css/style.css]: Your design has margins that are not present in the template design, please remove them.

Screen Shot 2022-06-29 at 18 23 00

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @samlorlah ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Clean codebase
- [x] Good job overall following the template design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (samlorlah) [reply index.html]: Done

File: asset/css/style.css (hunk 2 above)
Comment (samlorlah) [reply asset/css/style.css]: Fixed

File: index.html (hunk 3 above)
Comment (samlorlah) [reply index.html]: Done

File: asset/css/style.css (hunk 4 above)
Comment (samlorlah) [reply asset/css/style.css]: Removed

**Review by sumairq — APPROVED** [review]
### Hi @samlorlah  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**